SU2_CFD/src/solvers/CIncEulerSolver.cpp

```diff
@@ -462,6 +479,18 @@ void CIncEulerSolver::SetNondimensionalization(CConfig *config, unsigned short i
 
       case INC_IDEAL_GAS:
         fluidModel = new CIncIdealGas(Specific_Heat_CpND, Gas_ConstantND, Pressure_ThermodynamicND);
+        fluidModel->SetTDState_T(Temperature_FreeStreamND);        
+        break;
+
+      case FLUID_MIXTURE:
+        fluidModel = new CFluidScalar(Specific_Heat_CpND, Gas_ConstantND, Pressure_ThermodynamicND,config);
+        n_scalars  = config->GetnSpecies();
+        dummy_scalar = new su2double[n_scalars]();
+        for(int iVar=0;iVar<n_scalars; iVar++){
+      	    dummy_scalar[iVar] = config->GetSpecies_Init()[iVar];
+        }
```

Comment:
Can't you use the result of GetSpecies_Init directly without the extra copy?

Reply:
Yes, Now they are used directly